indico/modules/events/timetable/legacy.py

```diff
@@ -51,7 +51,7 @@ def serialize_timetable(self, event, days=None, hide_weekends=False):
                 continue
             if not entry.can_view(session.user):
                 continue
-            data = self.serialize_timetable_entry(entry)
+            data = self.serialize_timetable_entry(entry, load_children=False)
```

Comment:
I think it would be better to always serialize timetable entries with their children (session blocks only) and simply retrieve the top-level entries on this query.